src/libsync/creds/oauth.cpp

```diff
@@ -457,17 +480,25 @@ void OAuth::fetchWellKnown()
         OC_ASSERT(!urls.second.isNull());
         _authEndpoint = QUrl::fromUserInput(urls.first);
         _tokenEndpoint = QUrl::fromUserInput(urls.second);
+
+        qCDebug(lcOauth) << "override URL set, using auth endpoint" << _authEndpoint << "and token endpoint" << _tokenEndpoint;
```

Comment:
hmm not an override its more that the other values are the fallback

Reply:
I'm not sure I understand you right. This `if` handles the case when the theme enforces some URLs.

Reply:
Sorry parsed that wrong